samples/publish_workbook.py

```diff
@@ -41,7 +42,7 @@ def main():
     logging.basicConfig(level=logging_level)
 
     # Step 1: Sign in to server.
-    tableau_auth = TSC.TableauAuth(args.username, password)
+    tableau_auth = TSC.TableauAuth(args.username, password, site_id=args.sitename)
```

Comment:
args.sitename needs to also change to args.site

Reply:
Tested and Updated!